app/src/utils/conversions.ts

```diff
@@ -0,0 +1,19 @@
+import crypto from 'crypto'
```

Comment:
I'd rename this file `crypto.ts` or even better, `pocket.ts`. Conversions sounds like a math-related things and we're doing a very specific, protocol-related operation here.

On the other hand, I'm copying this straight into my scripts file!